clang/lib/AST/ASTDumper.cpp

```diff
@@ -361,3 +361,11 @@ LLVM_DUMP_METHOD void ConceptReference::dump(raw_ostream &OS) const {
   ASTDumper P(OS, Ctx, Ctx.getDiagnostics().getShowColors());
   P.Visit(this);
 }
+
+//===----------------------------------------------------------------------===//
+// Attr method implementations
+//===----------------------------------------------------------------------===//
+LLVM_DUMP_METHOD void Attr::dump() const {
+  ASTDumper P(llvm::errs(), /*ShowColors=*/false);
```

Comment:
Small nit: do we need an overload taking an `ASTContext` parameter to dump colors? Just like that on line 361.

Reply:
I think we normally just have a separate `dumpColor()` overload irrc, but I can add that yeah.

Reply:
I’ve removed this from this pr.